# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_file_protocol.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_file_protocol.js
--- a/netwerk/test/unit/test_file_protocol.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_file_protocol.js	Tue Feb 12 16:08:25 2019 +0000
@@ -141,7 +141,7 @@
   }
 
   chan.contentType = special_type;
-  chan.asyncOpen2(new FileStreamListener(on_read_complete));
+  chan.asyncOpen(new FileStreamListener(on_read_complete));
 }
 
 function do_test_read_dir(set_type, expected_type) {
@@ -163,7 +163,7 @@
 
   if (set_type)
     chan.contentType = expected_type;
-  chan.asyncOpen2(new FileStreamListener(on_read_complete));
+  chan.asyncOpen(new FileStreamListener(on_read_complete));
 }
 
 function test_read_dir_1() {
@@ -222,7 +222,7 @@
   }
 
   chan.contentType = special_type;
-  chan.asyncOpen2(new FileStreamListener(on_upload_complete));
+  chan.asyncOpen(new FileStreamListener(on_upload_complete));
 }
 
 function test_load_replace() {